src/AvaTaxClient.cs

```diff
@@ -354,13 +364,24 @@ private async Task<string> RestCallStringAsync(string verb, AvaTaxPath relativeP
             if (cd == null) cd = new CallDuration();
 
             // Convert the JSON payload, if any
-            string jsonPayload = null;
-            if (content != null) {
+            object jsonPayload = null;
+            string mimeType = null;
+
+            if(content != null && content is FileResult) {
```

Comment:
I did not see these changes in the Generator code. Are these not generated?

Reply:
AvaTaxClient.cs is a hand-written portion of the .NET 4.5 SDK. I believe the AvaTaxClient class in all the SDKs is manually written and maintained.

Reply:
Okay, thanks for the info.